### ISSUE TITLE
Sprite scale and rotation not updating hit boxes

### ISSUE THREAD

**Issue by MrGallo** [pvcraven/arcade#548]
The texture draws with the modified scale correctly, however, the hit-box points are not updated in `Sprite._set_scale()` as they are in `Sprite.__init__()`.

Example code:
```python
class Turret(arcade.Sprite):
    def __init__(self, image, center_x, center_y, angle):
        super().__init__(image)
        self.center_x = center_x
        self.center_y = center_y
        self.angle = angle
        self.scale = 0.4
        self.activated = False
```
I was able to toss the missing code in and it seemed to work. Unable to work on a PR at the moment.

```python
    def _set_scale(self, new_value: float):
        """Sets the scale of a sprite"""
        if new_value != self._scale:
            self.clear_spatial_hashes()
            self._point_list_cache = None
            self._scale = new_value
            if self._texture:
                self._width = self._texture.width * self._scale
                self._height = self._texture.height * self._scale
                self._texture.scale = self._scale  # ADDED

                points = self._texture.unscaled_hitbox_points  # ADDED
                scaled_points = [[value * self._scale for value in point] for point in points]  # ADDED
                self._points = scaled_points  # ADDED

                self.add_spatial_hashes()

            for sprite_list in self.sprite_lists:
                sprite_list.update_position(self)

```

**Comment by pvcraven** [comment]
Ah cool, thanks for the report.

**Comment by pvcraven** [comment]
There's a 2.2.3b1 out. There's other import changes which I need some more testing with.

**Comment by pvcraven** [comment]
Fixed in 2.2.5 out on 2020-Jan-17.
